ui/components/multichain/select-action-modal/select-action-modal.js

```diff
@@ -7,14 +7,9 @@ import {
   useLocation,
   ///: END:ONLY_INCLUDE_IF
 } from 'react-router-dom';
-import {
-  Box,
-  IconName,
-  Modal,
-  ModalContent,
-  ModalHeader,
-  ModalOverlay,
-} from '../../component-library';
+import { Box, IconName, Modal, ModalOverlay } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import { SelectActionModalItem } from '../select-action-modal-item';
 ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
 import useRamps from '../../../hooks/experiences/useRamps';
```

Comment:
Screenshots showing no visual changes to UI. @NidhiKJha it seems that the `ModalHeader` is missing padding is this intentional?

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 4 56 44 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d2c8b03a-5409-4d18-883f-f494a2783787">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 4 56 57 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/d20384a6-0582-4fee-b651-4c45d00cd67a">